provisioning/vars/tooling_version.yml

```diff
@@ -20,7 +20,7 @@ tooling:
     kube_linter: "0.2.5"
     # renovate: datasource=github-releases depName=helm/helm
     helm: "v3.8.1"
-    kubectl: "v1.23.0"
+    kubectl: "v1.26.4"
```

Comment:
Bin mir gerade unsicher:
backward kompatitbiltät ist gegeben? In prod (qwikinow) haben wir derzeit noch k8s 1.24 am laufen.
Bitte einmal gegenprüfen.